TicTacToe/ViewController.swift

```diff
@@ -0,0 +1,174 @@
+    var count = 0
+    var checkArray : [[Int]] = [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
+    var win = false
+    let numberOfRowColumn = 3
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor(patternImage:UIImage(named:"background.jpg")!)
+        firstPlayerLabel.text = firstPlayerName
+        secondPlayerLabel.text = secondPlayerName
+        commentLabel.text = "it's \(firstPlayerName)'s turn"
+        ticCollectionView.layer.borderColor = UIColor.grayColor().CGColor
+        ticCollectionView.layer.borderWidth = 2
+        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
+        layout.sectionInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+        layout.headerReferenceSize = CGSizeZero;
+        layout.footerReferenceSize = CGSizeZero;
+        layout.itemSize = CGSize(width: ticCollectionView.frame.size.width / CGFloat(numberOfRowColumn), height:ticCollectionView.frame.size.height / CGFloat(numberOfRowColumn))
+        layout.minimumLineSpacing = 0
+        layout.minimumInteritemSpacing = 0
+        ticCollectionView.collectionViewLayout = layout;
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
+    func numberOfSectionsInCollectionView(collectionView: UICollectionView) -> Int {
+        return numberOfRowColumn;
+    }
+
+    func collectionView(collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return numberOfRowColumn;
+    }
+
+    func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCellWithReuseIdentifier("TicCellIdentifier", forIndexPath: indexPath) as! TicCollectionViewCell
+        cell.backgroundColor = UIColor.grayColor()
+        cell.ticImageView?.image = ticArray[indexPath.row]
+        return cell
+    }
+
+    func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath: NSIndexPath) {
+        let cell : TicCollectionViewCell = collectionView.cellForItemAtIndexPath(indexPath) as! TicCollectionViewCell
+        checkArray[indexPath.section][indexPath.row] = currentPlayer
+        let toImage = currentPlayer == firstPlayer ? UIImage(named:"po.jpg") : UIImage(named:"jerry.jpg")
+        UIView.transitionWithView(cell.ticImageView, duration:0.4, options: UIViewAnimationOptions.TransitionFlipFromRight, animations: {
+            collectionView.userInteractionEnabled = false
+            cell.userInteractionEnabled = false
+            cell.ticImageView.image = toImage},
+                                  completion:{ (value:Bool) in
+                                    self.win = self.checkForWin(indexPath.section, column: indexPath.row, playerNumber: self.currentPlayer,path:indexPath)
```

Comment:
you can skip `self.` keyword for the class properties 
please follow everywhere you use this keyword